### ISSUE TITLE
Appearance of @import in comment breaks subsequent @font-face

### ISSUE THREAD

**Issue by mattdistefano** [jakubpawlowicz/clean-css#626]
This is on 3.3.6.

```css
/* this is a comment that mentions @import */
@font-face {
  font-family: "Font";
  src: url("path/to/font"); }
```

Is minified to: 

```css
@font-face{font-family:url(Font);src:url(path/to/font)}
```

Note that if the `@import` is a real `@import url();` that's just been commented out, it seems to minify fine.

Seems similar to #590.

**Comment by jakubpawlowicz** [comment]
So I've just run this on 3.3.6:

```bash
$ echo '/* this is a comment that mentions @import url("test.css"); */@font-face {font-family: "Font";src: url("path/to/font"); }' | cleancss
@font-face{font-family:Font;src:url(path/to/font)}
```

Do you use clean-css directly? It may be a more complicated case.

**Comment by mattdistefano** [comment]
Well this is interesting...

test.css:

```css
/* this is a comment that mentions @import */
@font-face {
  font-family: "Font";
  src: url("path/to/font"); }
```

Piping into cleancss produces the expected result:

```
type test.css | node ..\node_modules\clean-css\bin\cleancss
@font-face{font-family:Font;src:url(path/to/font)}
```

But passing the file name as an argument gives the erroneous output (FYI this is the same behavior I get through the gulp plugin):

```
node ..\node_modules\clean-css\bin\cleancss test.css
@font-face{font-family:url(Font);src:url(path/to/font)}
```

Note that the comment has to be just `@import` - if it's `@import url()` both of the above work fine.

**Comment by jakubpawlowicz** [comment]
Would you be able to share the file (my email is in package.json)?

**Comment by pascalstr** [comment]
Any updates on this @jakubpawlowicz ?

It works with @import url()  because of this regex: /^@import\s+(url\(|__ESCAPED)/i inside reduce.js

We tested this with clean-css directly and always ran into this problem. Using Gulp there is a difference in packages: With gulp-cssmin it works correctly and seems to correct the bug in clean-css. With gulp-minify-css it does not work correctly.

**Comment by jakubpawlowicz** [comment]
@pascalstr Unfortunately I wasn't able reproduce it. Any chance to share a reproducible test case? Can be an excerpt of a CSS file.

**Comment by mattdistefano** [comment]
Sorry, I haven't had a chance to send over the test file. It's literally just the snippet I included in the earlier comment, though. FYI this is on Windows 8 - haven't tried yet on OS X.

**Comment by pascalstr** [comment]
@jakubpawlowicz as @mattdistefano mentions earlier:

#####Step 1 create a test.css file with the following content:

```css
/* this is a comment that mentions @import */
@font-face {
  font-family: "Font";
  src: url("path/to/font"); 
}
```

#####Step 2 Then run clean-css 

node ..\node_modules\clean-css\bin\cleancss test.css

**Comment by jakubpawlowicz** [comment]
:+1: I can see it now!

**Comment by maciekpaprocki** [comment]
It's also happening on Mac. Had to remove import, from myfonts file to not brake build. It's not really edge case as most of the font selling websites force you to use @import to get counters of views.

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.4.
